### DISCUSSION TITLE
Hawk-eye Feature for Scanning Specific Columns in the Table

### DISCUSSION THREAD

**Discussion by 0xPb1** [rohitcoder/hawk-eye#3] [Ideas]
I am curious if Hawk-eye has the capability to scan specific columns. It would be challenging to scan two identical columns where one is important and the other is not, yet both contain similar or the same data. One of my friends recently faced an issue.

For example, various types of IDs with 10 digits and mobile numbers with 10 digits.

If this feature is not available, perhaps we could implement an `exclude_column` keyword to specify columns to exclude during the scan. Alternatively, we could add columns under the `tables:` field to specify the columns that you want the tool to scan.

 ``` mysql:
    mysql_example:
      host: YOUR_MYSQL_HOST
      port: YOUR_MYSQL_PORT
      user: YOUR_MYSQL_USERNAME
      password: YOUR_MYSQL_PASSWORD
      database: YOUR_MYSQL_DATABASE_NAME
      limit_start: 0   # Specify the starting limit for the range
      limit_end: 500   # Specify the ending limit for the range
      tables:
        - table1
        exclude_columns:
          - column1
          - column2
        - table2```

**Comment by rohitcoder** [comment]
Done, Solved as part of https://github.com/rohitcoder/hawk-eye/commit/80d8344a3d9769af94933dc4a0f30ed770c7a601 commit
